homeassistant/components/notify/pushover.py

```diff
@@ -64,13 +88,86 @@ def send_message(self, message='', **kwargs):
         if not isinstance(targets, list):
             targets = [targets]
 
+        file = {}
+
+        if file_data is not None:
+            filename = self.load_file(
+                url=file_data.get(ATTR_FILE_URL),
+                local_path=file_data.get(ATTR_FILE_PATH),
+                username=file_data.get(ATTR_FILE_USERNAME),
+                password=file_data.get(ATTR_FILE_PASSWORD),
+                auth=file_data.get(ATTR_FILE_AUTH))
+
+            if filename is not None:
+                file = (filename, open(filename, "rb"), "image/jpeg")
```

Comment:
Is this hard-coding the mime type to jpeg? Are other image mime types supported?

Reply:
Yes it is being hardcoded here. It was my understanding that the Pushover API only supports jpeg, but now that I look back at it, I don't see them specifying which image types they support (every example they show is a jpeg though, but that isn't definitive).

https://pushover.net/api#attachments

You raise a good point though, so I'll test their API directly with some other image types and see if this needs to be modified.

Reply:
Ok, great catch - looks like other types work as well.  I specifically tested GIF, JPEG, and PNG and all went through fine.  I'll remove the hardcoding and use mimetypes to guess it instead.